userguide/sharing.rst

```diff
@@ -1299,14 +1300,15 @@ for more details.
    sure to configure **all** SMB shares this way. Reboot the Mac
    client after making this change.
 
-These VFS objects do not appear in the drop-down menu and are dynamically
-configured:
+These VFS objects do not appear in the selection box:
 
 * **recycle:** moves deleted files to the recycle directory instead of
-  deleting them. Controlled by a checkbox in the share configuration.
+  deleting them. Controlled by :guilabel:`Export Recycle Bin`.
```

Comment:
Good!  It should still point the reader to where that checkbox is, so:

Controlled by :guilabel:`Export Recycle Bin` in the :ref:`SMB share configuration <smb_share_opts_tab>`.